src/ucp/core/ucp_request.h

```diff
@@ -186,7 +186,7 @@ struct ucp_request {
 
                         struct {
                             struct {
-                                void           *user_ptr;
+                                void           *ptr;
```

Comment:
pls document what this ptr could be in different cases (user ptr or am mpool element)